RecoTracker/LST/plugins/alpaka/LSTModulesDevESProducer.cc

```diff
@@ -0,0 +1,54 @@
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/ESProducer.h"
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/ModuleFactory.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+
+#include "RecoTracker/LST/interface/LSTModulesDev.h"
+#include "RecoTracker/Record/interface/TrackerRecoGeometryRecord.h"
+
+// LST includes
+#include <SDL/ModuleMethods.h>
+#include <SDL/LST.h>
```

Comment:
`LST.h` includes `Event.h` and now `Event.h` includes both `Module.h` and `ModuleMethods.h`. Hopefully that doesn't "break" the factorization you wanted to apply (probably that is more important for the `LSTModulesDev.h` file) but I guess it makes the first included unneeded here.

Reply:
the decoupling was for the device-agnostic header part in the interface